client/clientservice/src/main.cpp

```diff
@@ -30,6 +30,13 @@ using concord::client::concordclient::ConcordClientConfig;
 
 namespace po = boost::program_options;
 
+const static int kLogConfigRefreshIntervalInMs = 60 * 1000;
+
+const static char* GetLog4CplusConfigLocation() {
```

Comment:
Let's make this more user-friendly. If the environment variable isn't set then let's print a warning to `std::cerr`. Also, function names start with lower-case.

Reply:
Done.